### PR TITLE
doc: Improve Docs + add branding

### PR DESCRIPTION
**Pull request by daytime-em**
Uploading the resulting docs pages will be done by GitHub Action, as there is no reasonable way to do it via Gradle

### CODE REVIEW

**Comment by daytime-em** [comment]
This won't build until the gradle plugin has been completed

File: library/Module.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Module Mux Upload SDK
+
+The Mux Upload SDK processes and uploads video files to [Mux Video](https://www.mux.com/) from a
+user's local
+device. It is part of a full-stack flow described in our
+guide, [Upload Files Directly](https://docs.mux.com/guides/video/upload-files-directly).
```
Comment (refactornator) [inline library/Module.md:6]:
```suggestion
The Mux Upload SDK processes and uploads video files to [Mux Video](https://www.mux.com/video) from a user's local device. It is part of a full-stack flow described in our guide, [Upload Files Directly](https://docs.mux.com/guides/video/upload-files-directly).
```

File: library/Module.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  } else {
+    /*...*/
+  }
+}
+```
+
+## Resuming uploads after process death
+
+Uploads managed by this SDK can be resumed after process death, or if network connectivity caused 
+them to fail at some time in the past.
```
Comment (refactornator) [inline library/Module.md:49]:
```suggestion
Uploads managed by this SDK can be resumed after process death, or if network connectivity caused them to fail at some time in the past.
```

Comment (daytime-em) [reply library/Module.md:49]: The markdown renders without the line breaks, so this is kind of expected

File: library/Module.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+val upload = MuxUpload.Builder(myUploadUrl, myInputFile).build()
+upload.setResultListener { /*...*/ }
+upload.setProgressListener { /*...*/}
+upload.start()
+```
+
+### Handling errors
+
+The upload SDK handles transient errors according to a customizable retry policy. Fatal errors are
+reported by `MuxUpload.setResultListener`.
```
[the remark this answers is not in the document]
Comment (daytime-em) [reply library/Module.md:34]: This is rendered without the line breaks. To get a paragraph break in the markdown, you need two newlines.

**Review by refactornator — APPROVED** [review]
Looks great 🎉

**Pull request merged**